docs/core/resource.rst

```diff
@@ -198,7 +198,7 @@ As an example, let's build a code that will copy a collection into another:
     from pyramid import Configurator
 
 
-    config = Configurator(settings=DEFAULT_SETTINGS)
+    config = Configurator(settings={**DEFAULT_SETTINGS})
```

Comment:
Our default_settings aren't nested so yes I think it is enough.
Maybe we don't need to update the documentation about it.
The bug in Pyramid will probably be merged on Sunday but at first they said it was a feature and not a bug...